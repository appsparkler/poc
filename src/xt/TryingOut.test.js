import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import {TrialComponent} from './TryingOut.react'
import {configure, shallow} from 'enzyme'
import renderer from 'react-test-renderer'

configure({adapter: new Adapter()})

describe('Trial Component', () => {
  it('should render correctly', () => {
    const shallowWrapper = shallow(<TrialComponent msg="hey" />)
    // renderer.create(<TrialComponent msg="hey" />).toJSON()
    // console.log(shallowWrapper.html())
    expect(shallowWrapper.html()).toMatchSnapshot()
    shallowWrapper.setProps({msg: null})
    // expect(shallowWrapper.html()).toMatchInlineSnapshot(
    //     `"<div><h1>No Message</h1></div>"`
    // )
    expect(shallowWrapper.html()).toMatchSnapshot()
    // expect(shallowWrapper.find('.icon-star').length).toBe(1)
  })
})
