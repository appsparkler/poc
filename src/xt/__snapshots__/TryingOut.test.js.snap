// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trial Component should render correctly 1`] = `"<div><h1 class=\\"icon-star\\">Hello World.....hey</h1></div>"`;

exports[`Trial Component should render correctly 2`] = `"<div><h1>No Message</h1></div>"`;
